bob_report.csv

```diff
@@ -0,0 +1,170 @@
+VERSION,x86_64-macos
+1.2.38,0
+1.2.39,0
+1.2.40,0
+1.2.41,0
+1.2.42,0
+1.2.43,0
+1.2.44,0
+1.2.45,0
+1.2.46,0
+1.2.47,0
+1.2.48,0
+1.2.49,0
+1.2.50,0
+1.2.51,0
+1.2.52,0
+1.2.53,0
+1.2.54,0
+1.2.55,0
+1.2.56,0
+1.2.57,0
+1.2.58,0
+1.2.59,0
+1.2.60,0
+1.2.61,0
+1.2.62,0
+1.2.63,0
+1.2.64,0
+1.2.65,0
+1.2.66,0
+1.2.67,0
+1.2.68,0
+1.2.69,0
+1.2.70,0
+1.2.71,0
+1.2.72,0
+1.2.73,0
+1.2.74,0
+1.2.75,0
+1.2.76,0
+1.2.77,0
+1.2.78,0
+1.2.79,0
+1.2.80,0
+1.2.81,0
+1.2.82,0
+1.2.83,0
+1.2.84,0
+1.2.85,0
+1.2.86,0
+1.2.87,0
+1.2.88,0
+1.2.89,0
+1.2.90,0
+1.2.91,0
+1.2.92,0
+1.2.93,0
+1.2.94,0
+1.2.95,0
+1.2.96,0
+1.2.97,0
+1.2.98,0
+1.2.99,0
+1.2.100,0
+1.2.101,0
+1.2.102,0
+1.2.103,0
+1.2.104,0
+1.2.105,0
+1.2.106,0
+1.2.107,0
+1.2.108,0
+1.2.109,0
+1.2.110,0
+1.2.111,0
+1.2.112,0
+1.2.113,0
+1.2.114,0
+1.2.115,0
+1.2.116,0
+1.2.117,0
+1.2.118,0
+1.2.119,0
+1.2.120,0
+1.2.121,0
+1.2.122,0
+1.2.123,0
+1.2.124,0
+1.2.125,0
+1.2.126,0
+1.2.127,0
+1.2.128,0
+1.2.129,0
+1.2.130,0
+1.2.131,0
+1.2.132,0
+1.2.133,0
+1.2.134,0
+1.2.135,0
+1.2.136,0
+1.2.137,0
+1.2.138,0
+1.2.139,0
+1.2.140,0
+1.2.141,0
+1.2.142,0
+1.2.143,0
+1.2.144,0
+1.2.145,0
+1.2.146,0
+1.2.147,0
+1.2.148,0
+1.2.149,0
+1.2.150,0
+1.2.151,0
+1.2.152,0
+1.2.153,0
+1.2.154,0
+1.2.155,0
+1.2.156,0
+1.2.157,0
+1.2.158,0
+1.2.159,0
+1.2.160,0
+1.2.161,0
+1.2.162,0
+1.2.163,0
+1.2.164,0
+1.2.165,0
+1.2.166,107064052
+1.2.167,107078765
+1.2.168,107086636
+1.2.169,107288572
+1.2.170,135803529
+1.2.171,135803529
+1.2.172,135597602
+1.2.173,133654536
+1.2.174,133654536

```

Comment:
In all these reports, we have unfortunate gaps where the artefacts didn't upload correctly.
For small gaps, I just copy the same value.
At next generation, it will flip back to 0